packages/format-currency/src/number-format.js

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
Would be nice to have a CHANGELOG in this package where we'd document this change before we release the package. See https://github.com/Automattic/wp-calypso/blob/fix/reduce-packages-format-currency-deps/docs/monorepo.md#L170 for more info.

Reply:
If there were already a changelog I'd add an entry, but since there's not... 🤷 I could add one, but it'd lack information on everything else that has gone into the package.

Reply:
Feel free to introduce it, and for any prior changes, 🤷  indeed. It's always a good call to start somewhere 😉